Angular 6 / RXJS 6 Compatibility

**I'm submitting a ...**

[x] bug report
[ ] feature request
[ ] question about the decisions made in the repository

**Do you want to request a feature or report a bug?**

Bug

**What is the current behavior?**
Build Errors when upgrading to Angular 6 / RXJS 6

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem via
https://plnkr.co or similar (you can use one of the templates above as a starting point).**
upgrade rxjs to version ^6.0.0(-beta.2)

**What is the expected behavior?**
building without errors

**What is the motivation / use case for changing the behavior?**
keeping up to date with the new angular versions

**Please tell us about your environment:**
Angular version: 6.0.0-rc.0
Browser: all

> ERROR in node_modules/angular2-hotkeys/src/cheatsheet.component.d.ts(2,10): error TS2305: Module '"[...]/node_modules/rxjs/Subscription"' has no exported member 'Subscription'.
node_modules/angular2-hotkeys/src/hotkeys.service.d.ts(3,10): error TS2305: Module '"[...]/node_modules/rxjs/Subject"' has no exported member 'Subject'.


The Upgrade Guide can be found here: https://github.com/ReactiveX/rxjs/blob/master/doc/pipeable-operators.md

--- a/package.json
+++ b/package.json
@@ -39,7 +39,7 @@
     "@angular/core": "4.2.5",
     "@types/core-js": "0.9.41",
     "codelyzer": "~3.0.1",
-    "rxjs": "^5.4.0",
+    "rxjs": "^5.5.0",
     "tslint": "^3.15.1",
     "typescript": "~2.3.3",
     "zone.js": "^0.8.4"

--- a/src/cheatsheet.component.ts
+++ b/src/cheatsheet.component.ts
@@ -1,5 +1,5 @@
 import {Component, OnInit, OnDestroy, Input} from '@angular/core';
-import {Subscription} from 'rxjs/Subscription';
+import {Subscription} from 'rxjs';
 import {HotkeysService} from './hotkeys.service';
 import {Hotkey} from './hotkey.model';
 

--- a/src/hotkeys.service.ts
+++ b/src/hotkeys.service.ts
@@ -1,5 +1,5 @@
 import {HotkeyOptions, IHotkeyOptions} from './hotkey.options';
-import {Subject} from 'rxjs/Subject';
+import {Subject} from 'rxjs';
 import {Inject, Injectable} from '@angular/core';
 import {Hotkey} from './hotkey.model';
 import 'mousetrap';

